src/neuroconv/datainterfaces/behavior/video/externalvideointerface.py

```diff
@@ -87,13 +104,21 @@ def get_metadata_schema(self):
             "required": [self.video_name],
             "additionalProperties": True,
         }
+        metadata_schema["properties"]["Behavior"]["required"].append("ExternalVideoDevices")
+        metadata_schema["properties"]["Behavior"]["properties"]["ExternalVideoDevices"] = {
+            "type": "object",
+            "properties": {self.device_name: device_metadata_schema},
+            "required": [self.device_name],
+            "additionalProperties": True,
+        }
         return metadata_schema
 
     def get_metadata(self):
         metadata = super().get_metadata()
         video_metadata = {
             "Behavior": {
-                "ExternalVideos": {self.video_name: dict(description="Video recorded by camera.", unit="Frames")}
+                "ExternalVideos": {self.video_name: dict(description="Video recorded by camera.", unit="Frames")},
+                "ExternalVideoDevices": {self.device_name: dict(description="Video camera used for recording.")},
```

Comment:
I think this should be nested inside the metadata of the corresponding video to avoid collisions and have one less dynamic element on the metadata.